From 13f9ec24a6279a1e67efa1da4c73c9023525f70d Mon Sep 17 00:00:00 2001
From: Meng Mingming <mingming.meng@nxp.com>
Date: Wed, 27 Dec 2017 10:11:36 +0800
Subject: [PATCH 3173/5242] MLK-17311-2 gpu: imx: imx8_dprc: No need to round
 up height to 64

commit  d39357db563ae56592b8998e0f92e36c36c2e35a from
https://source.codeaurora.org/external/imx/linux-imx.git

It's necessary to make DPR baddr lie on the alignment block,
but no need to round up height to 64.

Signed-off-by: Meng Mingming <mingming.meng@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/imx8_dprc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/imx/imx8_dprc.c b/drivers/gpu/imx/imx8_dprc.c
index 1aedc8f..6afaf6b 100644
--- a/drivers/gpu/imx/imx8_dprc.c
+++ b/drivers/gpu/imx/imx8_dprc.c
@@ -353,7 +353,7 @@ void dprc_configure(struct dprc *dprc, unsigned int stream_id,
 			p1_w = round_up(width, info->cpp[0] == 2 ? 32 : 16);
 			break;
 		}
-		p1_h = round_up(height, modifier ? 64 : 4);
+		p1_h = round_up(height, 4);
 	}
 
 	dprc_write(dprc, PITCH(stride), FRAME_CTRL0);
-- 
1.7.9.5

